README.md

```diff
@@ -78,22 +78,24 @@ And buyers to:
 
  ### Local
 
+1. Fork the repo on GitHub
+
 1. Clone
-```
-git clone https://github.com/OriginProtocol/platform origin-platform && cd origin-platform
-```
+    ```
+    git clone git@github.com:YOUR_GITHUB_USERNAME/origin-js.git && cd origin-js
```

Comment:
Yeah, good point about `ssh` vs `https`. Now that you point it out, I think it's best not to assume what method the user will use to clone. I'm thinking we should just change this to say: 

`git clone` this repo

Or something like that.

Also, thanks for updating the reference to the repo!